src/game/client/components/hud.cpp

```diff
@@ -546,7 +546,7 @@ void CHud::RenderConnectionWarning()
 {
 	if(Client()->ConnectionProblems())
 	{
-		const char *pText = Localize("Connection Problems...");
+		const char *pText = Localize("Connection lost...");
```

Comment:
Yikes changing a very iconic message. It even has the wide spread acronym "cp" used by the community.

Reply:
Oh, sorry. I've only been playing for 2 years...